src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Ssl.cs

```diff
@@ -579,7 +582,7 @@ public static bool SslCheckHostnameMatch(SafeSslHandle handle, string hostName,
             // this code could be removed.
             //
             // It was verified as supporting case invariant match as of 10.12.1 (Sierra).
-            string matchName = new System.Globalization.IdnMapping().GetAscii(hostName);
+            string matchName = s_idnMapping.GetAscii(hostName);
```

Comment:
I think the comment above is invalid (`Since it's not documented as being thread safe, create a new one each time`).

We might also want to document that it is thread-safe if properties are not changed cc: @dotnet/docs.

Reply:
> We might also want to document that it is thread-safe if properties are not changed

Yes, I think we should.

Reply:
> I think the comment above is invalid

I'll delete it.